app/cafe.py

```diff
@@ -0,0 +1,23 @@
+import datetime
+
+import app.errors as errors
+
+
+class Cafe:
+    def __init__(self, name):
+        self.name = name
+
+    def visit_cafe(self, visitor: dict):
+        if 'vaccine' not in visitor:
+            raise errors.NotVaccinatedError(
+                f"Visitor {visitor['name']} has no vaccine"
+            )
+        if not visitor['vaccine']['expiration_date'] >= datetime.date.today():

```

Comment:
You can write `val1 < val2` instead of `not val1 >= val2`
It'd be more readable